resources/lang/en/profile.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
 return [
+	"browse" => [
+		"nousers" => "We don't seem to have any users at the moment",
```

Comment:
If I can be really picky, I would rather user underscore on the name, so it make it much more readable.